src/main/java/htsjdk/samtools/util/IOUtil.java

```diff
@@ -97,6 +96,11 @@
 
     private static int compressionLevel = Defaults.COMPRESSION_LEVEL;
 
+    /** signature of a gzip stream used by {@link isGZIPInputStream} 
```

Comment:
I think there are a few issues with the javadoc that my IDE pointed out to me.

 1.  to make that link work you need it to be `{@link #isGZIPInputStream}`
 2.  the `@see` doesn't need a `{@linkplain}`, it should be `* @see "http://stackoverflow.com/questions/4818468/how-to-check-if-inputstream-is-gzipped"` I believe.